### PR TITLE
Fix issue #258 - add ability to use clients without plugins

### PR DESCRIPTION
**Pull request by mshavliuk**
| Q                | A
| ---------------- | -----
| Bug fix          | yes
| New feature      | no
| BC breaks        | no
| Deprecations     | no
| Tests pass       | yes
| Fixed tickets    | #258
| License          | MIT

### CODE REVIEW

File: src/DependencyInjection/EightPointsGuzzleExtension.php [hunk 1]
```diff
@@ -150,6 +150,9 @@ protected function createHandler(ContainerBuilder $container, string $clientName
         }
 
         foreach ($this->plugins as $plugin) {
+            if (!isset($options['plugin'][$plugin->getPluginName()])) {
```
Comment (florianpreusner) [inline src/DependencyInjection/EightPointsGuzzleExtension.php]: I would do it the other way round: only execute "loadForClient" if value "isset".

Comment (mshavliuk) [reply src/DependencyInjection/EightPointsGuzzleExtension.php]: Yes, I agree. It will be better.

**Comment by mshavliuk** [comment]
As I see, all jobs are failed. I think if you add
```xml
<env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```
in your phpunit.xml, all these deprecation messages will go away

**Comment by mshavliuk** [comment]
Ok, so now I merged CI fix from origin/master (which named as trueOrigin in my system). Now it must pass all CI checks and become able to merge.

**Comment by florianpreusner** [comment]
Thanks for your contribution, @mshavliuk 👍 🎉

**Pull request merged**